src/cmd/cli/main.go

```diff
@@ -77,10 +77,11 @@ var rootCmd = &cobra.Command{
 			return nil
 		}
 
+		var filePath, _ = cmd.InheritedFlags().GetString("file")
 		cluster, _ := cmd.Flags().GetString("cluster")
 		nonInteractive, _ := cmd.Flags().GetBool("non-interactive")
 		provider, _ := cmd.Flag("provider").Value.(*cliClient.Provider)
-		client, tenantId = cli.Connect(cluster, *provider)
+		client, tenantId = cli.Connect(cluster, filePath, *provider)
```

Comment:
This is a bit weird. There's some commands like `secret` and `tail` that don't use a compose file, but would need a project name.

Reply:
Good point, I think for secret and tail the commands probably should have different behaviour when there is and there is not a valid compose file.

#### Secrets
Related to resource naming, when there is a compose file that has a project name that is different from the tenantID (not default project), the project name should be prepend to the secret name (/defang/project/secret), otherwise, the secret should be a default secret which only have defang prefix (/defang/secrent)

#### Tail
When there is a project name, tail only the logs for the current project, otherwise tail logs for all projects